lib/node_modules/@stdlib/math/base/special/absgammalnf/package.json

```
@@ -68,5 +68,87 @@
     "lgamma",
     "lgammaf",
     "float"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "absgammaln",
+      "alias": "absgammalnf",
+      "pkg_desc": "evaluate the natural logarithm of the absolute value of the gamma function",
+      "desc": "evaluates the natural logarithm of the absolute value of the gamma function",
+      "short_desc": "natural logarithm of absolute gamma function",
+      "parameters": [
+        {
+          "name": "x",
+          "desc": "input value",
+          "type": {
+            "javascript": "number",
+            "jsdoc": "number",
+            "c": "float",
+            "dtype": "float32"
+          },
+          "domain": [
+            {
+              "min": "-infinity",
+              "max": "infinity"
+            }
+          ],
+          "rand": {
+            "prng": "random/base/uniform",
+            "parameters": [
+              -10,
+              10
+            ]
+          },
+          "example_values": [
+            64,
+            27,
+            0,
+            0.1,
+            -9,
+            8,
+            -1,
+            125,
+            -10.2,
+            11.3,
+            -12.4,
+            3.5,
+            -1.6,
+            15.7,
+            -16,
+            17.9,
+            -188,
+            19.11,
+            -200,
+            21.15
+          ]
+        }
+      ],
+      "output_policy": "same",
+      "returns": {
+        "desc": "natural logarithm of absolute gamma function",
+        "type": {
+          "javascript": "number",
+          "jsdoc": "number",
+          "c": "float",
+          "dtype": "float32"
+        }
+      },
+      "keywords": [
+        "absgammalnf",
+        "gamma",
+        "log",
+        "logarithm",
+        "natural",
+        "absolute",
+        "special",
+        "function",
+        "single",
+        "precision"
+      ],
+      "extra_keywords": [
+        "math.lgamma"
+      ]
```

```suggestion
      "extra_keywords": []
```

There is no `lgamma` property on the built-in JavaScript `math` object.